staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -359,10 +425,20 @@ func (cfgCtl *configController) lockAndDigestConfigObjects(newPLs []*fctypesv1a1
 
 	meal.finishQueueSetReconfigsLocked()
 
+	if cfgCtl.embargoTime != nil && len(meal.fsStatusUpdates) == 0 && !meal.suppressedWrites {
+		klog.V(5).Infof("%s clearing embargo because no more need to update FlowSchemaStatus", meal.cfgCtl.name)
+		cfgCtl.embargoTime = nil
+	}
+
 	// The new config has been constructed
 	cfgCtl.priorityLevelStates = meal.newPLStates
-	klog.V(5).Infof("Switched to new API Priority and Fairness configuration")
-	return meal.fsStatusUpdates
+	if meal.suppressedWrites {
+		delay := DurationMax((*cfgCtl.embargoTime).Sub(cfgCtl.clock.Now()), time.Millisecond)
```

Comment:
is `delay` the same as `wait` above?  if so, naming consistency please.

Reply:
yes, renamed.